docs/tutorial/variables/variables.md

```diff
@@ -57,6 +57,7 @@ x =  5
 y =  "John"  
 print(type(x))  
 print(type(y))
+# Returns int and string respectivley
```

Comment:
```suggestion
# Returns number and string respectively
```
Currently `type` function returns `number` for both `int` and `float`.
```py
print (type(100)) # => number
print (type(10.4)) # => number
```
That's how JavaScript works. But python is different
```py
print (type(100)) # => int
print (type(10.4)) # => float
```
I don't have a decision 😭.

Reply:
I would say Javascript , I feel float and int is more precise as a return type , it's harder to narrow down the type of a 'number'. 🤷 It could make the compiler/transpiler  easier to make?